From 4adebef9de8293ffeee5f49a403393a143ebadd0 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@nxp.com>
Date: Wed, 8 Aug 2018 16:00:17 +0800
Subject: [PATCH 4410/5242] MLK-19168-6: ASoC: imx-rpmsg: update the rpmsg
 codec name

commit  60462b781302fbd28d957c8e01a304fafa16b28d from
https://source.codeaurora.org/external/imx/linux-imx.git

update the rpmsg codec name

Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
(cherry picked from commit 6130b919df97b127fafe319f0c2ff601ebc6bed2)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 sound/soc/fsl/imx-rpmsg.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/sound/soc/fsl/imx-rpmsg.c b/sound/soc/fsl/imx-rpmsg.c
index a5bb68d..7648e88 100644
--- a/sound/soc/fsl/imx-rpmsg.c
+++ b/sound/soc/fsl/imx-rpmsg.c
@@ -61,9 +61,9 @@ static int imx_rpmsg_probe(struct platform_device *pdev)
 
 	data->dai[0].name = "rpmsg hifi";
 	data->dai[0].stream_name = "rpmsg hifi";
-	if (rpmsg_i2s->codec) {
+	if (rpmsg_i2s->codec_wm8960) {
 		data->dai[0].codec_dai_name = "rpmsg-wm8960-hifi";
-		data->dai[0].codec_name = "rpmsg-audio-codec";
+		data->dai[0].codec_name = "rpmsg-audio-codec-wm8960";
 	} else {
 		data->dai[0].codec_dai_name = "snd-soc-dummy-dai";
 		data->dai[0].codec_name = "snd-soc-dummy";
-- 
1.7.9.5

